packages/pds/src/sequencer/events.ts

```diff
@@ -151,6 +167,10 @@ export const invalidatePrevHandleOps = async (db: Database, did: string) => {
   return invalidatePrevSeqEvts(db, did, ['handle'])
 }
 
+export const invalidatePrevOps = async (db: Database, did: string) => {
+  return invalidatePrevSeqEvts(db, did, ['append', 'handle', 'rebase'])
```

Comment:
enumerating all the different event types here feels sort of bad and prone to breaking next time an event type is added. i guess i could change the db query in `invalidatePrevSeqEvts` to just not have a `where eventType =` clause for this case? thoughts?